docker-compose.test.yml

```diff
@@ -1,21 +1,31 @@
 version: '3.8'
 services:
-    service-base: # rename this to something like <yourservicename>-service
-        build: .
+    feedback-portal:
+        build:
+            context: .
+            target: BASE_IMAGE
         volumes:
             - ./src:/usr/app/src
         ports:
             - '3000:3000'
+        volumes:
+            - ./src:/usr/app/src
+            - ./package.json:/usr/app/package.json
+            - ./yarn.lock:/usr/app/yarn.lock
+            - ./.babelrc:/usr/app/.babelrc
+            - ./.env:/usr/app/.env
         command: yarn test
         ###### uncomment below as appropriate
-        # depends_on:
-        #     - mongo
+        depends_on:
+            - mongo
         #     - postgresql
         #     - rabbitmq
         #     - redis
     ###### uncomment the correponsing db's/services you need
-    # mongo:
-    #     image: mongo:4.2.8-bionic
+    mongo:
+        image: mongo:4.2.8-bionic
+        ports:
+            - 27018:27018
```

Comment:
This is `host port:exposed port`.  Mongo should expose 27017 by default I believe (unless they just expose all ports on this container).  So all you would actually need to change is the host port.  Ex. `3005:27017` would map the exposed port 27017 on the mongo container to host port 3005.  I'm not sure that those ports currently would work.